src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_PRICE = 40;
+  const DISCONT_1 = 20;
+  const DISCONT_2 = 50;
```

Comment:
The variable name 'DISCONT_2' is not clear. Consider using a more descriptive name, like 'DISCOUNT_LONG_TERM'.